source/retryable-writes/tests/README.rst

```diff
@@ -171,6 +171,10 @@ Each YAML file has the following keys:
 
   - ``description``: The name of the test.
 
+  - ``skipReason`` (optional): A string describing why the test should be
```

Comment:
This was required so we could skip some bulkWrite tests due to https://jira.mongodb.org/browse/SERVER-53186.